docs/source/irene.rst

```diff
@@ -1,5 +1,193 @@
+ * ``/PMAPS/S1Pmt``: the sliced individual PMT waveforms for each S1 peak. 4 columns: event number, peak number, pmt id and amplitude (pes)
+ * ``/PMAPS/S2``: the sliced PMT-summed waveform for each S2 peak. 4 columns: event number, peak number, time (:math:`\mu`\ s) and amplitude (pes)
+ * ``/PMAPS/S2Pmt``: the sliced individual PMT waveforms for each S2 peak. 4 columns: event number, peak number, pmt id and amplitude (pes)
+ * ``/PMAPS/S2Si``: the sliced individual SiPM waveforms for each S2 peak. 4 columns: event number, peak number, sipm id and amplitude (pes)
+ * ``/Filters/empty_pmap``: flag for whether an event passed the empty pmap filter
+ * ``/Filters/s12_indices``: flag for whether an event passed the s12 indices filter
+
+.. _Irene config:
+
+Config
+------
+
+Besides the :ref:`Common arguments to every city`, *Irene* accepts the following arguments:
+
+.. list-table::
+   :widths: 60 40 120
+   :header-rows: 1
+
+   * - **Parameter**
+     - **Type**
+     - **Description**
+
+   * - ``n_baseline``
+     - ``int``
+     - Number of waveform samples to compute the baseline.
+
+   * - ``n_mau``
+     - ``int``
+     - Number of waveform samples to compute the mau.
+
+   * - ``thr_mau``
+     - ``float``
+     - Threshold for mau calculation in pes.
+
+   * - ``thr_sipm``
+     - ``float``
+     - Threshold for individual SiPM samples. Can be absolute (pes) or relative (unitless), depending on ``thr_sipm_type``.
+
+   * - ``thr_sipm_type``
+     - ``ThresholdSiPM``
+     - Thresholding mode for individual SiPM samples. ``common`` applies the same absolute threshold value to all SiPMs. ``individual`` uses a relative value based on the noise spectrum for each SiPM.
+
+   * - ``s1|s2_lmin|lmax``
+     - ``int``
+     - Lower/upper limits to the width of S1/S2 signals expressed in number of samples.
+
+   * - ``s1|s2_tmin|tmax``
+     - ``float``
+     - Lower/upper limits of the search window for S1/S2 signals.
+
+   * - ``s1|s2_rebin_stride``
+     - ``int``
+     - Rebin factor for S1/S2 signals. Rarely changed. 1 for S1 and 40 for S2 signals.
+
+   * - ``s1|s2_stride``
+     - ``int``
+     - Allowed range of signal fluctuations below threshold for peak merging expressed in number of samples.
+
+   * - ``thr_csum_s1|s2``
+     - ``float``
+     - Threshold applied to the PMT-summed waveform in order to find S1/S2 peaks.
+
+   * - ``thr_sipm_s2``
+     - ``float``
+     - Threshold applied to the time-integrated signal of each SiPM to discard SiPMs with only dark counts.
+
+   * - ``pmt_samp_wid``
+     - ``float``
+     - Sampling period of PMTs. Should be removed.
+
+   * - ``sipm_samp_wid``
+     - ``float``
+     - Sampling period of SiPMs. Should be removed.
+
+
+.. _Irene workflow:
+
+Workflow
+--------
+
+Irene performs a number of data transformations in order to obtain a **PMap**. These operations can be grouped in four main tasks:
```

Comment:
The first two don't need to follow that specific order, but the implementation follows that sequence. Apart from that, the order needs to be that one because each step relies on the transformations of the previous ones.

Reply:
I would then specify that the city follows that sequence, for compliteness.